test/BUILD.bazel

```
@@ -46,8 +45,12 @@ cc_library(
     ],
 )
 
-cc_test(
-    name = "eventuals",
+cc_library(
+    name = "eventuals-tests",
```

I'd like to suggest `base` here to mimic the naming in `eventuals/BUILD.bazel`:
```suggestion
    name = "base",
```